uaa/src/test/java/org/cloudfoundry/identity/uaa/orchestrator/UaaOrchestratorClaimTest.java

```diff
@@ -0,0 +1,301 @@
+    // https://user-management-dev.pss.dev.usw02.15.energy, https://user-management-stg.pss.dev.usw02.15.energy
+    private static String UMS_EKS_ENV_URI_REG =
+            "(https://[-a-zA-Z0-9]*(\\.uaa-qa\\.pss\\.dev\\.usw02\\.15\\.energy){1,1})";
+
+    @Before
+    public void setupDataBeforeEachTestRun() {
+        uaaRequest = getUaaRequest();
+    }
+
+    // region Positive Tests
+    @Test
+    public void testCreateZone() throws Exception {
+
+        OrchestratorZoneResponse getZone = null;
+        String zoneName = null;
+        String claimName =
+                (UAA_CLAIM_NAME_PREFIX +/* testConfig.getTestRunIdentifier() + */ "-" + (++uaaInstanceCount)).toLowerCase();
+
+        Claim claim = null;
+        boolean isClaimCreated = false;
+        boolean isZoneCreated = false;
+        try {
+            isClaimCreated = createZone(claimName, uaaRequest);
+            Thread.sleep(120000);
+            claim = getZone(claimName, ServiceName.UAA);
+            assertNotNull(claim);
+            assertEquals(claimName, claim.getMetadata().getName());
+
+            zoneName = namespace + "-" + claimName;
+            String getResponse = getZoneByName(zoneName, HttpStatus.OK);
+            getZone = OBJECT_MAPPER.readValue(getResponse, OrchestratorZoneResponse.class);
+
+            assertNotNull(getResponse);
+            assertNotNull(getZone);
+            assertEquals(zoneName, getZone.getName());
+            assertNotNull(getZone.getConnectionDetails().getDashboardUri());
+            assertNotNull(getZone.getConnectionDetails().getUri());
+            assertNotNull(getZone.getConnectionDetails().getIssuerId());
+            assertEquals(uaaRequest.getSubdomain(), getZone.getConnectionDetails().getSubdomain());
+
+            JsonNode outputSecret = orchestratorUtil.readSecret(claimName, ServiceName.UAA);
+            String outputSecretData = StringEscapeUtils.unescapeJava(outputSecret.toString());
+            assertNotNull(outputSecret);
+            assertNotNull(outputSecretData);
+
+            String uaaEksUri = null;
+            Pattern pattern = Pattern.compile(UMS_EKS_ENV_URI_REG);
+            Matcher matcher = pattern.matcher(outputSecretData);
+            if (matcher.find()) {
+                uaaEksUri = matcher.group();
+            }
+            assertTrue(outputSecretData.contains(uaaEksUri));
+
+            // if claim and outputSecret(created by Orchestrator as result of successful instance creation) both are not null
+            // that means uaa zone is successfully created in uaaz service.
+
+            UaaForProvider uaaforProvider = ((UaaClaimSpec) claim.getSpec()).getUaaForProvider();
+
+            assertEquals(uaaRequest.getSubdomain(), uaaforProvider.getSubdomain());
+            assertNotNull(outputSecret.get("connectionDetails"));
+
+        } catch (Exception e) {
```

Comment:
why empty catch block, remove it if not needed or log exception.

Reply:
Added for debugging purpose Removed IT